SUPPORT.md

```diff
@@ -10,16 +10,13 @@
 
 # Support
 
-## How to file issues and get help  

```

Comment:
This whole file is just a placeholder, @rabollin you guys checking in a template. Please read through and fill it out as needed

Reply:
sure